stubs/docutils/docutils/nodes.pyi

```diff
@@ -1,6 +1,3 @@
-from typing import Any, List
-
-class reference:
-    def __init__(self, rawsource: str = ..., text: str = ..., *children: List[Any], **attributes: Any) -> None: ...
+from typing import Any
 
 def __getattr__(name: str) -> Any: ...  # incomplete
```

Comment:
for my own knowledge, does this `__getattr__` act as a catch-all for undefined objects or something?

Reply:
Yes, this means that every name can be imported from `docutils.nodes` and is treated as `Any`. So `docutils.nodes.reference` is `Any` for now, so it should at least report no wrong errors. Of course it would be better to have proper type annotations here.